test/e2e/e2e_test.go

```diff
@@ -248,6 +248,32 @@ var _ = Describe("Zero Trust Workload Identity Manager", Ordered, func() {
 			By("Waiting for SPIFFE CSI Driver DaemonSet to become Available")
 			utils.WaitForDaemonSetAvailable(testCtx, clientset, utils.SpiffeCSIDriverDaemonSetName, utils.OperatorNamespace, utils.DefaultTimeout)
 		})
+
+		It("SPIRE OIDC provider installation and readiness can be verified", func() {
+			By("Creating SpireOIDCDiscoveryProvider object")
+			spireOIDCDiscoveryProvider := &operatorv1alpha1.SpireOIDCDiscoveryProvider{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "cluster",
+				},
+				Spec: operatorv1alpha1.SpireOIDCDiscoveryProviderSpec{
+					TrustDomain: appDomain,
+					JwtIssuer:   fmt.Sprintf("https://oidc-discovery.%s", appDomain),
```

Comment:
Considering `JwtIssuer` is required in both `SpireServer` and `SpireOIDCDiscoveryProvider` objects, we could define it as a global variable in `BeforeAll` block. Just like what we did for appDomain:

https://github.com/openshift/zero-trust-workload-identity-manager/blob/ade5c5d0c7293b2b4c728a6d30022adb2c192b5f/test/e2e/e2e_test.go#L38

https://github.com/openshift/zero-trust-workload-identity-manager/blob/ade5c5d0c7293b2b4c728a6d30022adb2c192b5f/test/e2e/e2e_test.go#L48

This would lower the chance of unintentional misconfiguration